cmd/kube-proxy/app/server.go

```diff
@@ -421,6 +422,8 @@ func createClients(config componentconfig.ClientConnectionConfiguration, masterO
 
 // Run runs the specified ProxyServer.  This should never exit (unless CleanupAndExit is set).
 func (s *ProxyServer) Run() error {
+	// To help debugging, immediately log version
```

Comment:
this comment is not helpful.

Reply:
yeah, but I'd like to leave it as-is to keep consistent with what #46047 did :)